lib/Support/CMakeLists.txt

```diff
@@ -21,6 +34,18 @@ add_circt_library(CIRCTSupport
   MLIRIR
   )
 
+
+add_circt_library(CIRCTSupportTestPasses
```

Comment:
I _think_ the policy is to put this in the `tests` directory, but I'm not sure. Would make sense given that it is a test.

Reply:
Thats what I did for the esi api test and the C API tests are that way too.